### PR TITLE
[BUGFIX beta] IE10-8 fixes. normalizeProperty expects lowercase prop name

### PR DESCRIPTION
**Pull request by mixonic**
Mostly extracted from https://github.com/emberjs/ember.js/pull/10186. IE9+ is all good, IE8 getting closer.

### CODE REVIEW

**Comment by mixonic** [comment]
I've labelled this bugfix beta, though I'm unsure how smoothly it will merge onto beta.

File: packages/ember-views/lib/views/text_field.js [hunk 1]
```diff
@@ -54,6 +54,8 @@ export default Component.extend(TextSupport, {
     'width'
   ],
 
+  defaultLayout: null,
```
Comment (mixonic) [inline packages/ember-views/lib/views/text_field.js]: Need some feedback here. text fields are input tags. In IE8 an exception is raised if you try to append blank text nodes to an input.

text_field is now a component, so a layout makes sense. Not sure why it is being appended to the view element though.

Comment (rwjblue) [inline packages/ember-views/lib/views/text_field.js]: This does make sense.  `<input>` cannot have child elements, this is just enforcing that (and preventing errors in IE8).

Comment (mixonic) [inline packages/ember-views/lib/views/text_field.js]: Fwiw @rwjblue and I have discussed ignoring `defaultLayout` when there is no template or layout on a component.

**Pull request merged**